### ISSUE TITLE
Can't load kext after build

### ISSUE THREAD

**Issue by jonwest** [blankmac/AlpsT4USB#2]
I'm trying to build the latest commit from master, and get an error:
```
/Library/Extensions/AlpsT4USB.kext failed to load - (libkern/kext) link error; check the system/kernel logs for errors or try kextutil(8).
```

When I run `sudo kextutil AlpsT4USB.kext` I get:
```
Kext with invalid signatured (-67050) allowed: <OSKext 0x7f94ae50ed80 [0x7fff93bd58e0]> { URL = "file:///Library/Extensions/AlpsT4USB.kext/", ID = "com.blankmac.AlpsT4USB" }
Kext with invalid signatured (-67050) allowed: <OSKext 0x7f94ae50d580 [0x7fff93bd58e0]> { URL = "file:///Library/Extensions/VoodooI2C.kext/", ID = "com.alexandred.VoodooI2C" }
Kext with invalid signatured (-67050) allowed: <OSKext 0x7f94ae536c90 [0x7fff93bd58e0]> { URL = "file:///Library/Extensions/VoodooI2C.kext/Contents/PlugIns/VoodooI2CServices.kext/", ID = "com.alexandred.VoodooI2CServices" }
Kext with invalid signatured (-67050) allowed: <OSKext 0x7f94ae536b60 [0x7fff93bd58e0]> { URL = "file:///Library/Extensions/VoodooI2C.kext/Contents/PlugIns/VoodooGPIO.kext/", ID = "org.coolstar.VoodooGPIO" }
Code Signing Failure: code signature is invalid
Disabling KextAudit: SIP is off
(kernel) kxld[com.blankmac.AlpsT4USB]: The super class vtable '__ZTV17IOHIDEventService' for vtable '__ZTV20AlpsT4USBEventDriver' is out of date. Make sure your kext has been built against the correct headers.
(kernel) Can't load kext com.blankmac.AlpsT4USB - link failed.
(kernel) Failed to load executable for kext com.blankmac.AlpsT4USB.
(kernel) Kext com.blankmac.AlpsT4USB failed to load (0xdc008016).
(kernel) Failed to load kext com.blankmac.AlpsT4USB (error 0xdc008016).
Failed to load /Library/Extensions/AlpsT4USB.kext - (libkern/kext) link error.
Check library declarations for your kext with kextlibs(8).
```

I'm trying to build it by just running `git clone <url> .` and then `xcodebuild`, and the build succeeds--unfortunately this is my first attempt at building a kext, so I apologize if I'm missing something obvious here.

This is for a HP Elite x2 1012 G1.

Thanks!

**Comment by jonwest** [comment]
Going to close this for now--it's not working, but I'm not convinced this is an issue anywhere but behind the keyboard, so there's no reason to drag you into this.

**Comment by blankmac** [comment]
Sorry for the late reply, been a little bit busy lately.  I'm not able to recreate your problem.  What version of Xcode are you using and what OS version are you running?

**Comment by jonwest** [comment]
Oh, man--don't even sweat it at all, it's not your job to debug every random's problem just because you've gone to the trouble of making your repo public! 😂 

I ended up "fixing" this (but not actually).  For whatever reason (as I said, behind-the-keyboard error) xcodebuild was building against the Catalina SDK and I was running `xcodebuild` from Mojave.  I couldn't find where to get the Mojave SDK, and the "workaround" of just symlinking `MacOS` to `MacOS-10.14` (I forget the exact folder name) didn't work.

I ended up installing a fresh copy of Catalina and things built and loaded totally fine.  Had a few hiccups getting bluetooth working, but got it sorted out thanks to the posts in your guide.

Thanks for everything!
